### PR TITLE
further clarifications for li role mapping

### PR DESCRIPTION
**Pull request by scottaohara**
adds clarity that the `li` should be generic if not a child of one of the list elements, OR if the list element's list role has been suppressed / removed.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/pull/476.html" title="Last updated on May 19, 2023, 12:36 PM UTC (a8703d4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/476/fa9234e...a8703d4.html" title="Last updated on May 19, 2023, 12:36 PM UTC (a8703d4)">Diff</a>

### CODE REVIEW

**Review by stevefaulkner — APPROVED** [review]

**Comment by cookiecrook** [comment]
Somewhat related to https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/20

File: index.html [hunk 1]
```diff
@@ -2211,8 +2211,8 @@ <h3>HTML Element Role Mappings</h3>
               <td class="ax"><div class="general">Use WAI-ARIA mapping</div></td>
               <!-- <td class="naming"></td> -->
               <td class="comments">
-                If `li` element is not a child of <a data-cite="html">`ol`</a> , <a data-cite="html">`menu`</a> or <a data-cite="html">`ul`</a>
-                then expose the `li` element with a `generic` role.
+                If `li` element is not a child of <a data-cite="html">`ol`</a> , <a data-cite="html">`menu`</a> or <a data-cite="html">`ul`</a>, or if the containing 
+                list element is no longer exposed with a `list` role, then expose the `li` element with a `generic` role.
```
Comment (cookiecrook) [inline index.html:2178]: should this be a separate row to include a value for the new `computedrole` column?

Comment (scottaohara) [reply index.html:2178]: yes.  but i would do that further work in #465

Comment (cookiecrook) [reply index.html:2178]: 👌

**Review by cookiecrook — APPROVED** [review]

**Pull request merged**